include/knowhere/index/index_node.h

```diff
@@ -202,7 +203,7 @@ class IndexNode : public Object {
             return GenResultDataSet(nq, std::move(range_search_result));
         }
 
-        auto its_or = AnnIterator(dataset, std::move(cfg), bitset);
+        auto its_or = AnnIterator(dataset, std::move(cfg), bitset, false);
```

Comment:
Please add a comment about why this parameter is `false`

Reply:
added. 
> The range_search function has utilized the search_pool to concurrently handle various queries.
> To prevent potential deadlocks, the iterator for a single query no longer requires additional thread control over the next() call.